src/AliasingContainer.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Dhii\Container;
+
+use Dhii\Container\Util\StringTranslatingTrait;
+use Psr\Container\ContainerInterface;
+use function array_key_exists;
+
+/**
+ * A container implementation that wraps around an inner container to alias its keys, so consumers can use the aliases
+ * to fetch data from the inner container.
+ */
+class AliasingContainer implements ContainerInterface
```

Comment:
I generally implement the [Dhii `ContainerInterface`][1] in Dhii packages. the implementation is a bit more complex, because the forwarding is not as simple. But it has several advantages for consumers. Do you think it's worth it?

[1]: https://github.com/Dhii/data-container-interface/blob/develop/src/ContainerInterface.php

Reply:
Didn't understand the latter part about the "implementation".

Reply:
Don't remember what I was referring to by that either. But still think that implementing the Dhii interface could be better for ISP.

Reply:
The Dhii interface doesn't add anything that these classes need to implement, and using the lowest-level compatible interface is natural IMO.

Not like it changes anything for consumers, but curious to hear why. If you tell me that it's just a convention that you like to stick to for Dhii packages, I'll change it.

Reply:
No, not a convention. Dhii interfaces add extra features that could be useful. But I'm willing to just move on for now, and return to this point in the future if it comes up.